hatch_build.py

```diff
@@ -225,6 +225,8 @@
     "itsdangerous>=2.0",
     "jinja2>=3.0.0",
     "jsonschema>=4.18.0",
+    # Added for CLI
+    "keyring>=25.6.0",
```

Comment:
Do we recommend/require anywhere the native os backends for this?

Reply:
I have added the `keyring.alt` earlier for development purposes for the required part. This is only recommended for development. So this will be enough to test. Surely production use cases should be recommended. Closing this thread for now, I will tag you on the docs so we can continue there. Thanks again Ash for your time on this one!